src/Commands/config/ConfigExportCommands.php

```diff
@@ -3,28 +3,26 @@
 
 use Consolidation\AnnotatedCommand\CommandData;
 use Consolidation\AnnotatedCommand\Events\CustomEventAwareInterface;
+use Consolidation\AnnotatedCommand\Events\CustomEventAwareTrait;
 use Drupal\Core\Config\StorageComparer;
 use Drupal\Core\Config\FileStorage;
 use Drush\Config\StorageWrapper;
 use Drush\Commands\DrushCommands;
 
 class ConfigExportCommands extends DrushCommands implements CustomEventAwareInterface {
 
-  use ConfigTrait;
+  use CustomEventAwareTrait;
```

Comment:
We can now remove all lines that mention CustomEvent in this file. Same for config-import